tests/test_match_path.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from openapi.helpers import match_path
+from openapi.exceptions import NotFoundError
+
+
+class MatchPathTestCase(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.paths = (
+            '/pets',
+            '/pets/{pet_id}',
+        )
+
+    def test_simple_path(self):
+        path_match, path_parameters = match_path('/pets', self.paths)
+        self.assertEqual(path_match, '/pets')
+        self.assertEqual(path_parameters, {})
+
+    def test_parameter_path(self):
+        path_match, path_parameters = match_path('/pets/15', self.paths)
+        self.assertEqual(path_match, '/pets/{pet_id}')
+        self.assertEqual(path_parameters, {'pet_id': '15'})
+
+    def test_not_found_path(self):
+        with self.assertRaises(NotFoundError) as context:
```

Comment:
Any reason for an exception instead of returning `None` or `(None, None)`?

Reply:
I'm planning to use this lib as a quick way to validate web requests and responses against an OpenAPI specification. Raising a Not Found exception here will make it easier to capture it using some kind of exception middleware provided by web frameworks, for example.

Reply:
Of course, this is my objective now but it could change as I continue implementing the other parts of this lib.